content/install/linux.md

```diff
@@ -19,23 +22,27 @@ disable_comments: true
 
 [Install Stack](https://docs.haskellstack.org/en/stable/README/#how-to-install). Stack is a build tool for Haskell / Clash projects.
 
-_If you want to setup a Clash project with Cabal instead, read [the starter project's README.md](https://github.com/clash-lang/clash-starters/tree/main/simple#simple-starter-project)._
-
 # 3. Setup Clash
-Depending on your goals, you might want to simply run Clash on its own, or setup a proper project. The first one will get you up and running quickly and you won't have to learn build tools. However, you won't be able to use dependencies from [Hackage](https://hackage.haskell.org/) and it is harder to get a consistent build environment.
 
-### Option A. Run Clash on its own
-The following compiles a file `HelloWorld.hs` to VHDL:
+To setup a new project based on the provided starter projects, run:
 
 ```
-stack exec --resolver lts-19 --package clash-ghc -- clash HelloWorld.hs --vhdl
+stack new my-clash-project clash-lang/simple
 ```
 
-### Option B. Setup a starter project
-To setup a new project based on the provided starter projects, run:
+This will create a new project called `my-clash-project` in a folder named the same. The folder will contain a `README.md` to get you up and running. Alternatively, you [can read it online](https://github.com/clash-lang/clash-starters/tree/main/simple#simple-starter-project).
```

Comment:
```suggestion
This will create a new project called `my-clash-project` in a folder with the same name. The folder will contain a `README.md` to get you up and running. Alternatively, you [can read it online](https://github.com/clash-lang/clash-starters/tree/main/simple#simple-starter-project).
```